docs/Developers/Newcomers_Guide.rst

```diff
@@ -87,7 +87,41 @@ Step 1. Meet the Community!
 
 To get started, the first step is to meet the community. We use gitter to
 communicate, and there the helpful community will guide you.
-Join us at `coala gitter <https://coala.io/chat>`_.
+Gitter is an instant messaging service used by developers and users of GitHub.
+Gitter uses chatrooms, where developers can join in and can talk about a
+particular topic.
+coala has 2 types of chatrooms - repository chatrooms and discussion topics.
+Repository chatrooms are related to a specific repository and
+discussion chatrooms are related to general discussion topics like
+conferences, workshops, etc.
+
+  * `coala <https://gitter.im/coala/coala>`_
+    This is the main chatroom and repository chatroom of coala/coala.
+  * `gsoc <https://gitter.im/coala/coala/gsoc>`_
+    This is where you discuss about Google Summer of Code.
+  * `coala-bears <https://gitter.im/coala/coala-bears>`_
+    Repo chatroom for coala/coala-bears.
+  * `workshop <https://gitter.im/coala/coala/workshops>`_
+    Discussions related to workshops go here.
+  * `conferences <https://gitter.im/coala/conferences>`_
+    Everything related to conferences.
+  * `offtopic <https://gitter.im/coala/coala/offtopic>`_
+    Anything fun! Our gaming sessions start here.
+
+The list of all available chatrooms are available here - `channel list <https://coala.io/channels>`_
+
+But before joining the community, here are few things that you should
+keep in mind.
+
+  * Don't @-mention or private message people, unless its utterly important.
+    @ mentions generate notifications on the various gitter clients the user
+    may be signed into, you might even wake someone on the other side of the
+    world up. Also it discourages other people to answer the question,
+    so you might wait longer for an answer.
+  * Never use @/all as a newcomer.
```

Comment:
I'm pretty sure most of don't use /all often either. Feel like this could be rephrased to say something more like "Don't use /all if you are a newcomer or do not have a critical reason"
Thoughts?